LICENSE

```diff
@@ -1,3 +1,12 @@
+Copyright (c) 2021-present PrairieLearn, Inc.
+
+Portions of this software are licensed as follows:
+
+* All content that resides under the "ee/" directory of this repository, if that directory exists, is licensed under the license defined in "ee/LICENSE".
+* All client-side JavaScript (when served directly or after being compiled, arranged, augmented, or combined), is licensed under the "MIT Expat" license.
+* All third party components incorporated into the PrairieLearn Software are licensed under the original license provided by the owner of the applicable component.
+* Content outside of the above mentioned directories or restrictions above is available under the "GNU Affero General Public License" as defined below.
+
                     GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
We should also add the MIT license (https://opensource.org/licenses/MIT) at the bottom of this file, because under MIT we need to include a copy of the license text.

Reply:
Good call, will do.

Reply:
Done in e2e9bd0.